src/http/server/handlers/static_file_handler.cr

```diff
@@ -64,6 +64,14 @@ class HTTP::StaticFileHandler
       context.response.content_type = "text/html"
       directory_listing(context.response, request_path, file_path)
     elsif is_file
+      last_modified = File.stat(file_path).mtime.to_s("%a, %-d %h %C%y %T GMT")
```

Comment:
Please use `HTTP.rfc1123_date` as a standardized formatter (same in the specs). Your custom format is exactly the same as rfc1123 just without leading zero for the day. The leading zero is recommended by [RFC 7231](https://tools.ietf.org/html/rfc7231#section-7.1.1.1) which describes the `HTTP-date` format for the `Last-Modified` header in [RFC 7232](https://tools.ietf.org/html/rfc7232#section-2.2).